File: 009_Palindrome_Number/009_1 (str).java
class Solution {
    public boolean isPalindrome(int x) {
        if (x < 0) return false;

        String s = String.valueOf(x);
        String a = new StringBuffer(s).reverse().toString();
        if (s.equals(a))
            return true;
        else
            return false;
    }
}

File: 009_Palindrome_Number/009_1 (str).cpp
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;

class Solution {
public:
    bool isPalindrome(int x) {
        if (x < 0) return false;
        string s = to_string(x);
        string s2 = s;
        reverse(s2.begin(), s2.end());
        return s == s2;
    }
};


int main() {
    Solution s;
    int x = 13331;
    cout << boolalpha;
    cout << s.isPalindrome(x) << endl;
}